iotedgehubdev/cli.py

```diff
@@ -189,66 +199,59 @@ def modulecred(modules, local, output_file):
               help='Docker daemon socket to connect to')
 @_with_telemetry
 def start(inputs, port, deployment, verbose, host):
-    configFile = HostPlatform.get_config_file_path()
-    try:
-        with open(configFile) as f:
-            jsonObj = json.load(f)
-            if CONN_STR in jsonObj and CERT_PATH in jsonObj and GATEWAY_HOST in jsonObj:
-                connection_str = jsonObj[CONN_STR]
-                cert_path = jsonObj[CERT_PATH]
-                gatewayhost = jsonObj[GATEWAY_HOST]
-                edgeManager = EdgeManager(connection_str, gatewayhost, cert_path)
-                if host is not None:
-                    os.environ[DOCKER_HOST] = host
-            else:
-                output.error('Missing keys in config file. Please run `iotedgehubdev setup` again.')
-                sys.exit(1)
-    except Exception as e:
-        raise e
+    config_json = _load_config_json()
 
-    hostname_hash, suffix = Utils.hash_connection_str_hostname(connection_str)
-    telemetry.add_extra_props({'iothubhostname': hostname_hash, 'iothubhostnamesuffix': suffix})
+    if config_json:
+        connection_str = config_json[CONN_STR]
+        cert_path = config_json[CERT_PATH]
+        gatewayhost = config_json[GATEWAY_HOST]
+        edgeManager = EdgeManager(connection_str, gatewayhost, cert_path)
+        if host is not None:
+            os.environ[DOCKER_HOST] = host
 
-    if inputs is None and deployment is not None:
-        try:
-            with open(deployment) as json_file:
-                json_data = json.load(json_file)
-                if 'modulesContent' in json_data:
-                    module_content = json_data['modulesContent']
-                elif 'moduleContent' in json_data:
-                    module_content = json_data['moduleContent']
+        hostname_hash, suffix = Utils.hash_connection_str_hostname(connection_str)
```

Comment:
Turn on "Hide whitespace changes" and you can see that this section only indent the code. :(